chain-impl-mockchain/src/leadership/genesis/mod.rs

```diff
@@ -136,6 +136,137 @@ impl GenesisLeaderSelection {
     }
 }
 
+#[cfg(test)]
+mod test {
+
+    use super::*;
+    use crate::ledger::test::create_initial_fake_ledger;
+    use crate::milli::Milli;
+    use crate::stake::PoolStakeDistribution;
+    use crate::stake::StakePoolInfo;
+    use chain_addr::Discrimination;
+    use std::collections::HashMap;
+
+    fn make_pool(ledger: &mut Ledger) -> (StakePoolId, SecretKey<Curve25519_2HashDH>) {
+        let mut rng = rand::thread_rng();
+
+        let pool_vrf_private_key = SecretKey::generate(&mut rng);
+
+        let pool_info = StakePoolInfo {
+            serial: 1234,
+            owners: vec![],
+            initial_key: GenesisPraosLeader {
+                vrf_public_key: pool_vrf_private_key.to_public(),
+                kes_public_key: SecretKey::generate(&mut rng).to_public(),
+            },
+        };
+
+        let pool_id = pool_info.to_id();
+
+        ledger.delegation.register_stake_pool(pool_info).unwrap();
+
+        (pool_id, pool_vrf_private_key)
+    }
+
+    #[test]
+    #[ignore]
+    pub fn test_phi() {
+        let slots_per_epoch = 200000;
+        let active_slots_coeff = 0.1;
+
+        let (_genesis_hash, mut ledger) = create_initial_fake_ledger(
+            Discrimination::Test,
+            &vec![],
+            slots_per_epoch,
+            Milli::from_millis((active_slots_coeff * 1000.0) as u64),
+        );
+
+        let mut pools = HashMap::<StakePoolId, (SecretKey<Curve25519_2HashDH>, u64, Value)>::new();
+
+        let (big_pool_id, big_pool_vrf_private_key) = make_pool(&mut ledger);
+        pools.insert(
+            big_pool_id.clone(),
+            (big_pool_vrf_private_key, 0, Value(1000)),
+        );
+
+        for _i in 0..10 {
```

Comment:
you can just do `_` in Rust :) 

https://play.rust-lang.org/?code=fn%20main()%0A%7B%0A%20%20%20%20for%20_%20in%200..10%20%7B%0A%20%20%20%20%20%20%20%20println!(%22Hello%20World%22)%3B%0A%20%20%20%20%7D%0A%7D&version=stable

Reply:
That's not our code, tho )